### PR TITLE
Update dev people section

### PR DESCRIPTION
**Pull request by SamuelChua**
# Overview 

Addressed #685 "Feature request: Automate updating the people section on dev page"

By using the `officer-parser.mjs` script in `scripts`, it fetches the officer's data from this [spreadsheet](https://docs.google.com/spreadsheets/d/1-qx14c6smCjvvgAbKYO0oF17kwvbNIB_XskbOCUq3pM/edit#gid=0) and populate it in the dev team's page under `dev.js` automatically. `OfficerCard.js` was edited to update the input fields compared to the old `data/dev.js`

Changed image size from 300x300 to 220x220 due to new photo requirements and centralize photo vertically and horizontally as shown below

image
image
image

## Changes

`components/OfficerCard.js`
- Updated fields based on offput.json output generated from 'scripts/officer-parser.mjs'

`styles/components/OfficerCard.module.scss`
- Changed CSS styles in .officer-image-container from max-width : 300px to 220px
- Added justify-content: center to centralize the image 

`pages/dev.js`
- Updated Dev Team Director details
- Added devTeamOfficers variable which filters out based on officers who are in the Dev Team

### CODE REVIEW

File: scripts/officer-parser.mjs [hunk 1]
```diff
@@ -5,6 +5,7 @@ import * as dotenv from "dotenv";
 dotenv.config();
 const SERVICE_ACCOUNT = process.env.SERVICE_ACCOUNT;
 const DIRECTORY_SPREADSHEET_ID = process.env.DIRECTORY_SPREADSHEET_ID;
+console.log(SERVICE_ACCOUNT);
```
Comment (snigdha-kansal) [inline scripts/officer-parser.mjs:8]: Can we remove this line?

File: scripts/officer-parser.mjs [hunk 2]
```diff
@@ -19,6 +20,7 @@ async function getGoogleSheetData(range) {
   const sheets = google.sheets({ version: "v4" });
 
   // Get JWT Token to access sheet
+  console.log('SERVICE_ACCOUNT:', SERVICE_ACCOUNT);
```
Comment (snigdha-kansal) [inline scripts/officer-parser.mjs:23]: Can we remove this line as well?

File: scripts/officer-parser.mjs (hunk 1 above)
Comment (SamuelChua) [reply scripts/officer-parser.mjs]: done

File: scripts/officer-parser.mjs (hunk 2 above)
Comment (SamuelChua) [reply scripts/officer-parser.mjs]: done

**Review by snigdha-kansal — APPROVED** [review]
Looks great. Nice work, Samuel! We need to add the Advisor role since Officer Parser only reads from the Officers tab. Can make another PR for it.

**Pull request merged**